src/jupyblog/md.py

```diff
@@ -263,6 +264,7 @@ def render(self, name, *, include_source_in_footer):
         md_ast = self.parser(md_raw)
         # TODO: replace and use model object
         metadata = parse_metadata(md_raw)
+        metadata['jupyblog']['version'] = version()
```

Comment:
looks good; please add the "version" under a `metadata` section. so it's more explicit that this is an automated field that should not be edited by users

Reply:
I will do it during the day! thanks!